myapp/database/config.js

```diff
@@ -0,0 +1,18 @@
+var mysql = require('mysql');
```

Comment:
Why should we have to use two same files at different locations?
**myapp./database/config.js** and **myapp/database.js**?